### PR TITLE
Run tests on GitHub Actions

### LINKED ISSUE #25: Run tests on CI
Just to know, what do you think about running the automated tests on CI?

I can create a PR adding this feature if you'd like.

### PR DESCRIPTION
**Pull request by 64J0**
## Description:

With this PR, I'm adding the GitHub Actions configuration to run the tests automatically on CI, either when a PR is open/updated or when manually triggered.

Must close https://github.com/TheAlgorithms/F-Sharp/issues/25.

### CODE REVIEW

**Comment by 64J0** [comment]
Example running the workflow: https://github.com/64J0/TheAlgorithms-F-Sharp/pull/1

File: Algorithms.Tests/Strings/MinCostStringConversionTests.fs [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-        Assert.AreEqual(expected, actual)
-
+﻿namespace Algorithms.Tests.Strings
+
+open Microsoft.VisualStudio.TestTools.UnitTesting
+open Algorithms.Strings
+
+// FIXME
+// [<TestClass>]
+// type MinCostStringConversionTests () =
```
Comment (64J0) [inline Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: This is failing the tests, as explained on https://github.com/TheAlgorithms/F-Sharp/issues/25.

Comment (mertcandav) [reply Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: Can you fix this test?

Comment (64J0) [reply Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: I think I can. Found it written in Python too -> https://github.com/TheAlgorithms/Python/blob/master/strings/min_cost_string_conversion.py

Comment (64J0) [reply Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: I'm not really sure what to use for the test inputs.

Comment (64J0) [reply Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: This Python example is raising an exception too `System.ArgumentException: The index was outside the range of elements in the list. (Parameter 'n')`.

Comment (mertcandav) [reply Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: Alright. This PR is for running tests for CI. It does not require correction of the test. I'll merge your PR.
Please open an issue for this test issue.

**Comment by mertcandav** [comment]
Thanks for your contribution.

File: Algorithms.Tests/Strings/MinCostStringConversionTests.fs (hunk 1 above)
Comment (64J0) [reply Algorithms.Tests/Strings/MinCostStringConversionTests.fs:8]: Done -> https://github.com/TheAlgorithms/F-Sharp/issues/27

**Pull request merged**